hack/ci/e2e-k8s.sh

```diff
@@ -142,6 +154,10 @@ networking:
   kubeProxyMode: ${KUBE_PROXY_MODE:-iptables}
 nodes:
 - role: control-plane
+  extraMounts:
+  - hostPath: audit/
```

Comment:
I still think that you should keep at is and use the relative path here appending the artifacts folder
```suggestion
  - hostPath: ${ARTIFACTS}/audit/
```

Reply:
Scratch it, my fault